triagebot.toml

```diff
@@ -1170,9 +1169,7 @@ codegen = [
     "@saethlin",
     "@workingjubilee",
 ]
-query-system = [
-    "@cjgillot",
-]
+query-system = []
```

Comment:
What happens when a path in the sources has no one?

Reply:
I'm fine being added there. Not an expert, but I want to know when things are happening and can maintain a minimal quality